yarn.lock

```diff
@@ -2,6 +2,19 @@
 # yarn lockfile v1
```

Comment:
Is this supposed be here? I'm not 100% on how NUbook works but none of the other PRs have this file changed

Reply:
The `yarn.lock` file sets versions for your dependencies. It should be changed when you change your dependencies or their versions. More info [here](https://classic.yarnpkg.com/en/docs/yarn-lock/). 
Basically, good catch! It shouldn't be committed here.